packages/date-picker/test/fullscreen.test.js

```diff
@@ -108,20 +108,6 @@ describe('fullscreen mode', () => {
     });
   });
 
-  describe('focused attribute', () => {
-    it('should keep focused attribute after opening overlay', async () => {
-      datePicker.focus();
-      await open(datePicker);
-      expect(datePicker.hasAttribute('focused')).to.be.true;
-    });
-
-    it('should remove focused attribute when closing overlay', async () => {
```

Comment:
**note:** Since the `focused` attribute isn't set on small screens, it doesn't make sense to assert its absence on close.